### DISCUSSION TITLE
Update to new version

### DISCUSSION THREAD

**Discussion by durbanbr** [rustdesk/rustdesk-server#307] [General]
Hi,
I installed version 1.1.9 and would like to update to the new version, which has new features. What is the procedure to do this?

**Comment by dinger1986** [comment]
How did you install?

**Comment by durbanbr** [comment]
I installed it as it is on the website, using install.sh from the website.

wget https://raw.githubusercontent.com/techahold/rustdeskinstall/master/install.sh
chmod +x install.sh
./install.sh

**Reply by dinger1986** [reply]
Go to techaholds repo and use the upgrade script from there https://rustdesk.com/docs/en/self-host/rustdesk-server-oss/install/#how-to-install-the-server it even says here 👍

**Reply by durbanbr** [reply]
Sorry, I couldn't find the update script. Where can I get it? I couldn't find this link!

**Reply by dinger1986** [reply]
![image](https://github.com/rustdesk/rustdesk-server/assets/7244447/9d1d0611-2745-4ccb-b177-85d97e0dd200)

Literally scroll down a bit more from the link I gave you and it likes to techaholds repo

**Reply by durbanbr** [reply]
Thank you very much and sorry, it was easy to see and I couldn't see it!!

**Reply by dinger1986** [reply]
Sometimes easy to miss the things under our nose
